src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -691,6 +691,39 @@ public async Task<ActionResult<ApiResourceAllocationRequest>> ApproveProjectAllo
             return new ApiResourceAllocationRequest(result!);
         }
 
+        [HttpDelete("/resources/requests/internal/{requestId}/workflow")]
+        [HttpDelete("/departments/{departmentString}/resources/requests/{requestId}/workflow")]
+        public async Task<ActionResult> ResetWorkflow(Guid requestId, string? departmentString)
+        {
+            var requestItem = await DispatchAsync(new GetResourceAllocationRequestItem(requestId));
+
+            if (requestItem == null)
+                return ApiErrors.NotFound("Could not locate request", $"{requestId}");
+
+            #region Authorization
+
+            var authResult = await Request.RequireAuthorizationAsync(r =>
+            {
+                r.AlwaysAccessWhen().FullControl().FullControlInternal();
+                r.AnyOf(or =>
+                {
```

Comment:
Should anyone on the taskowner side be able to reset the workflow?

Reply:
I thought about this, but couldn't think of any use cases where that would be required and decided to put it off for now, since we can always add that in easily later, while it's a bit hard to think about the implications of letting the task owner side delete the workflow.